src/components/Footer/Footer.css

```
@@ -0,0 +1,25 @@
+.footer-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  color: #222421;
+}
+
+.footer-links {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-size: 1.5vw;
+}
+
+.footer-mobile-app, .footer-creators {
+  margin: 30px 30px 30px 60px;
+}
+
+.footer-copyright {
+  color: #767a74;
+  margin-right: 30px;
+  font-size: 1.5vw;
+}
```

Let's make sure there's a newline at the end of files.
```suggestion
}

```